src/app_service/service/chat/service.nim

```diff
@@ -186,6 +184,11 @@ QtObject:
       error "error: ", errDesription
       return
 
+  proc init*(self: Service) =  
+    self.doConnect()
+    self.loadChats()
```

Comment:
FYI This is refactored to be async in my PR here https://github.com/status-im/status-desktop/pull/9363